### PR TITLE
SOC-1505 | Displaying message to blocked user when opening editor

### PR DESCRIPTION
**Pull request by bkoval**
Original pull request - https://github.com/Wikia/mercury/pull/1739

https://wikia-inc.atlassian.net/browse/SOC-1505

### CODE REVIEW

File: front/scripts/main/models/discussion-forum.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default App.DiscussionForumModel = DiscussionBaseModel.extend(DiscussionD
 	pageNum: null,
 	posts: null,
 	totalPosts: 0,
+	isRequesterBlocked: true,
```
Comment (kvas-damian) [inline front/scripts/main/models/discussion-forum.js]: Do you think we should assume that user is blocked by default?

Comment (rafalkalinski) [inline front/scripts/main/models/discussion-forum.js]: +1 for the question

Comment (bkoval) [inline front/scripts/main/models/discussion-forum.js]: Yeah. If we don't get the sufficient response from the API we'll redirect to the error page with no editor anyway. Assuming false-negative in this case should be safe then.

**Comment by kvas-damian** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Comment by rafalkalinski** [comment]
https://github.com/Wikia/mercury/pull/1794

**Pull request closed without merging**